content/en/developers/ide_integrations/idea/_index.md

```diff
@@ -0,0 +1,150 @@
+- link: "/getting_started/profiler/"
+  tag: "Documentation"
+  text: "Getting started with Continuous Profiler."
+---
+
+{{< beta-callout url="#" btn_hidden="true">}}
+  The Datadog plugin for IntelliJ IDEA is in public beta. It is intended for Java developers who use the <a href="https://docs.datadoghq.com/profiler/#pagetitle">Continuous Profiler</a> for their Java services. If the plugin stops working unexpectedly, check for plugin updates or <a href=#feedback>reach out to the team</a>.
+{{< /beta-callout >}}
+
+## Overview
+
+The Datadog plugin for IntelliJ IDEA helps you to improve software performance by providing meaningful code-level insights in the IDE based on real-time observability data. Together with the Continuous Profiler, the plugin helps you to reduce latency and lower cloud costs by highlighting code lines that:
+- consume the most CPU
+- allocate the most memory
+- spend the most time on locks, disk I/O, socket I/O, and so on
+
+{{< img src="/developers/ide_integrations/idea/overview1.png" alt="The Datadog tool window open in IDEA" style="width:100%;" >}}
+
+In addition to code highlighting, you can use the Datadog plugin for Intellij IDEA to:
+- Identify methods that consume the most resources with the **Top List** view.
+- Visualize aggregated profile data with the **Flame Graph** view.
+- Navigate directly from the top list and flame graph to the relevant lines in your code.
+- Find resource consumption broken down by method name and line number.
+
+## Requirements
+
+- **A Datadog account**: The plugin requires a Datadog account. If you're new to Datadog, go to the [Datadog website][4] to learn more about Datadog's observability tools and sign up for a free trial.
+- **Continuous Profiling**: To display code-level insights, the plugin requires Continuous Profiling instrumented on your Java Services. For more information, see [Getting Started with the Continuous Profiler][3].
+
+## Setup
+
+### Install the Datadog plugin
+
+1. Click **Plugins** and search for `Datadog`.
+1. Click **Install** to download and install the plugin in your IDE.
+1. If you receive a prompt notifying you that Datadog is a third-party plugin, click **Accept**.
+1. Click **Restart IDE** to restart IDEA.
+
+{{< img src="/developers/ide_integrations/idea/datadog-plugin1.png" alt="The Datadog plugin" style="width:100%;" >}}
+
+### Add Datadog credentials
+
+After installing the Datadog plugin and restarting IDEA, add your Datadog API and application keys to authenticate with Datadog:
+1. With a file or project open in IDEA, click the **Datadog** tool window.
+1. Click **Add your credentials...**.
+1. Enter your name, [API key and application key][5], and select a site.
+
+**Note**: For most users, one pair of keys is all that is required. If you're using a multi-org setup with multiple key pairs, check to ensure that the correct pair is active using the **Tools** -> **Datadog** section in **Preferences**.
```

Comment:
a link might be useful

Reply:
This is inside the IDEA app, I can probably refine though 🤔